pmd-cataloger/src/main/java/sfdc/sfdx/scanner/pmd/catalog/PmdCatalogRule.java

```diff
@@ -23,15 +34,15 @@
 
 
   public PmdCatalogRule(Element element, PmdCatalogCategory category, String language) {
-    this.name = element.getAttribute("name");
-    this.message = element.getAttribute("message");
+    this.name = element.getAttribute(ATTR_NAME);
+    this.message = element.getAttribute(ATTR_MESSAGE);
     this.language = language;
     this.category = category;
     this.description = getDescription(element);
   }
 
   String getFullName() {
-    return getCategoryPath() + "/" + getName();
+    return getCategoryPath() + PATH_JOINER + getName();
```

Comment:
Traditionally (in java) this is called a path separator.  But wondering why you made it a constant?

Reply:
Had plans for adding a test for this, but ended up not doing that. Will change it back to what it was.